appender.go

```diff
@@ -54,9 +76,7 @@ func NewAppenderFromConn(driverConn driver.Conn, schema string, table string) (*
 		return nil, fmt.Errorf("can't create appender")
 	}
 
-	var chunkSize = C.duckdb_vector_size()
-
-	return &Appender{c: dbConn, schema: schema, table: table, appender: &a, currentRow: 0, currentChunkIdx: 0, currentChunkSize: chunkSize}, nil
+	return &Appender{c: dbConn, schema: schema, table: table, appender: &a, currentChunkSize: 0}, nil
```

Comment:
This is unrelated to your PR, but I noticed a memory leak in this function. Also, we do not return the respective error message.

```diff
@@ -62,21 +65,26 @@ func NewAppenderFromConn(driverConn driver.Conn, schema string, table string) (*
                panic("database/sql/driver: misuse of duckdb driver: Appender after Close")
        }
 
-       var schemastr *(C.char)
+       var schemaStr *C.char
        if schema != "" {
-               schemastr = C.CString(schema)
-               defer C.free(unsafe.Pointer(schemastr))
+               schemaStr = C.CString(schema)
+               defer C.free(unsafe.Pointer(schemaStr))
        }
 
-       tablestr := C.CString(table)
-       defer C.free(unsafe.Pointer(tablestr))
+       tableStr := C.CString(table)
+       defer C.free(unsafe.Pointer(tableStr))
 
-       var a C.duckdb_appender
-       if state := C.duckdb_appender_create(*dbConn.con, schemastr, tablestr, &a); state == C.DuckDBError {
-               return nil, fmt.Errorf("can't create appender")
+       var appender C.duckdb_appender
+       state := C.duckdb_appender_create(*dbConn.con, schemaStr, tableStr, &appender)
+
+       if state == C.DuckDBError {
+               // we'll destroy the error message when destroying the appender
+               err := errors.New(C.GoString(C.duckdb_appender_error(appender)))
+               C.duckdb_appender_destroy(&appender)
+               return nil, err
        }
 
-       return &Appender{c: dbConn, schema: schema, table: table, appender: &a, currentChunkSize: 0}, nil
+       return &Appender{c: dbConn, schema: schema, table: table, appender: &appender, currentChunkSize: 0}, nil
 }
```